docs/running-against-production.md

```diff
@@ -1,25 +1,62 @@
-## Running the client against production
+# Running the client against production
 
 If you want to run your local client against the production server, we have a way to do that using ngrok:
 
+## Set up `ngrok` to connect to the server
+
 - Join our ngrok account by asking Paul
 - install ngrok: `brew cask install ngrok` on OS X, or follow instructions at
   https://ngrok.com/download for Linux. (The snap installer is broken.)
 - authorize your ngrok client: `ngrok authtoken YOURTOKEN` (found at
   https://dashboard.ngrok.com/auth)
-- run your tunnel: `ngrok http 8000 --hostname=darklang-<username>.ngrok.io`
-- You can simplify this to `ngrok start darklang` by adding to your
+- run your tunnel:
+  `ngrok http darklang.localhost:8000 --hostname=darklang-<your-username>.ngrok.io`
```

Comment:
It would be worth mentioning here that your username should be your Darklang username -- it's intended to disambiguate between users so we don't run over each other's tunnels. It might also be a good idea to use some random digits as a suffix so people can't get access to our local servers by guessing our usernames